plugins/modules/cloudfront_distribution.py

```diff
@@ -1711,6 +1738,14 @@ def validate_origin(self, client, existing_config, origin, default_origin_path):
                 origin['custom_headers'] = ansible_list_to_cloudfront_list(origin.get('custom_headers'))
             else:
                 origin['custom_headers'] = ansible_list_to_cloudfront_list()
+            if 'origin_shield' in origin:
+                origin_shield = origin.get('origin_shield')
+                if origin_shield.get('enabled') is not None:
```

Comment:
You can replace the two first lines:
```python
if origin_shield.get('enabled') is not None:
    if origin_shield['enabled']:
```

With just:

`if origin_shield.get('enabled'):`

`.get()` will return `None` if the key is missing. Python sees both `None` and `False` as `False`.